sunpy/net/dataretriever/sources/tests/test_eve.py

```diff
@@ -155,8 +153,8 @@ def test_show(LCClient):
     mock_qr = mock_query_object(LCClient)
     qrshow0 = mock_qr.show()
     qrshow1 = mock_qr.show('Start Time', 'Instrument')
-    allcols = ['Start Time', 'End Time', 'Instrument', 'Physobs', 'Source',
-               'Provider', 'Level']
-    assert qrshow0.colnames == allcols
+    allcols = {'Start Time', 'End Time', 'Instrument', 'Physobs', 'Source',
+               'Provider', 'Level', 'url'}
```

Comment:
Is the url column a new feature that needs a changelog?

Reply:
It was always in the results, just now there is no difference between what the user sees in the results and what is in the results, where as before we only constructed a table with some of the columns for display. I think this is a general changelog entry rather than one specific to the url column.